client/src/main/java/com/vesoft/nebula/client/graph/ResultSet.java

```diff
@@ -33,7 +33,7 @@ public ResultSet() {
     public ResultSet(List<byte[]> columns, List<RowValue> rows) {
         this.columns = Lists.newArrayListWithCapacity(columns.size());
         for (byte[] column : columns) {
-            this.columns.add(new String(column));
+            this.columns.add(new String(column).intern());
```

Comment:
when you calling `intern`, if it in the const pool, it will return if not will append into the pool. 

Because the column maybe used many times, so this will take some improvement?

I don't know if I'm right

Reply:
you can try it
https://juejin.im/post/5d8871c76fb9a06b0703fcbe